packages/web/src/app/contexts/api/rates/hooks/useMarketsData/useMarketsData.ts

```diff
@@ -1,28 +1,47 @@
 import { useMemo } from 'react';
 import { useQueries } from 'react-query';
 
+import { useTokens } from '@p2p-wallet-web/core';
 import { zip } from 'ramda';
 
 import type { Market, Markets } from 'app/contexts';
+import { useConfig } from 'app/contexts';
 
 import { marketLoader } from './marketLoader';
 
-export const useMarketsData = (keys: (string | null | undefined)[]): Markets => {
+// eslint-disable-next-line @typescript-eslint/no-magic-numbers
+const THIRTY_SECONDS = 1000 * 30;
+
+export const useMarketsData = (symbols: (string | null | undefined)[]): Markets => {
+  const { tokenConfigs } = useConfig();
```

Comment:
You should not use useConfig(). It's legacy code for orca only, but this is a common hook for all pages

Reply:
Changed